### PR TITLE
Load shot image directly

### PR DESCRIPTION
**Pull request by 6a68**
TL;DR: moves shot image to a plain old image tag, making the shot page functional in IE, Edge, and any browser with JS disabled.

The shot image wasn't loading in Edge or IE (#2935) because those browsers don't support the CSP nonce directive. (The very latest version of Edge, released this week with the Windows 10 Creators Update, does support CSP level 2.) I'm not sure why the regular image loading detection code failed, but the CSP error explains why the inline fallback code failed.

This seemed like a good opportunity to get rid of the code that hides the image until it's loaded. Simplifies the code and makes the pages accessible without JS.

I set the spinner as a centered background image of the image tag itself; it's weird-looking now, but once jgruen replaces the svg with a simple png, should look good.

### CODE REVIEW

**Comment by ianb** [comment]
Looks like there's a conflict.

Do you want to land this as-is and let John fix it up, or have him add to this branch?

**Comment by ianb** [comment]
That was a through-the-web conflict resolution, so I haven't actually tested it.

**Comment by 6a68** [comment]
@ianb Rebased against current master, and it seems to be working in Edge. Mind taking another look?

File: server/src/pages/shot/view.js [hunk 1]
```diff
@@ -41,34 +35,16 @@ class Clip extends React.Component {
       console.warn("Somehow there's a shot without an image");
       return null;
     }
-    let node = <img id="clipImage" style={{height: "auto", width: clip.image.dimensions.x + "px", maxWidth: "100%", display: this.state.imageDisplay}} ref={clipImage => this.clipImage = clipImage} src={ clip.image.url } alt={ clip.image.text } onLoad = { this.onImageLoaded.bind(this) } />;
-    // Note that in server/src/pages/shot/page.js there is also JavaScript defined
-    // that displays the image onload, as a backup to make sure the image always
-    // gets displayed even if the bundle doesn't load
+    let backgroundStyle = "transparent url('" + this.props.staticLink("/static/img/spinner.svg") + "') center no-repeat";
```
Comment (ianb) [inline server/src/pages/shot/view.js]: This could go in a stylesheet, but eh, whatever
